src/darsia/image/patches.py

```diff
@@ -638,9 +636,8 @@ def blend_and_assemble(self, update_img: bool = False) -> darsia.Image:
         da_assembled_img = darsia.Image(
             img=assembled_img,
             origin=self.base.origin,
-            width=self.base.width,
-            height=self.base.height,
-            color_space=self.base.colorspace,
+            dimensions=self.base.dimensions,
+            color_space=self.base.color_space,
```

Comment:
This is not general. `Image` in general does not have color space e.g. In general one can just use `metadata()` in the `kwargs` of `Image`. Also instead of `Image`, I guess using `type` of the input will be most general to ensure the output type is identical to the input type.

Reply:
I think I updated this correctly now with the metadata (please have a look). I was however a bit unsure of what you mean by type. I'm not 100% sure how it works. Can you either explain quickly or show me the correct way (it sounds like its supposed to be easy and allow for more flexibility?).